dev/cp-configs-to-root

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Meant to be run from the project root
+
+cp config/.env* .
```

Comment:
There is almost twice as many chars in this filename as there is in the only command it's running :-) Seems like this script is superfluous.

Reply:
Heh. I guess this is more for my convenience while developing. Maybe I'll just keep it in my `config/` dir which is ignored by git. My one thought is, I find it useful, and maybe others would, too? If they area also keeping their test configs in the hidden `config/` dir.

Reply:
I can also make it have a shorter name, but I wanted it to be descriptive.

Reply:
Decided to remove the file. https://github.com/iFixit/pulldasher/pull/203/commits/4e08edc2ae6fed66535adfc0309c15835b65074d